{"kind":"symbol","identifier":{"url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/AssertionError(_:)","interfaceLanguage":"swift"},"metadata":{"roleHeading":"Case","role":"symbol","symbolKind":"case","title":"PythonError.AssertionError(_:)","modules":[{"name":"SwiftPy"}],"externalID":"s:7SwiftPy11PythonErrorO09AssertionD0yACSScACmF","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AssertionError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"AssertionError"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"platforms":["macOS"]}]}],"variants":[{"paths":["\/documentation\/swiftpy\/pythonerror\/assertionerror(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError"]]},"schemaVersion":{"major":0,"patch":0,"minor":3},"sections":[],"references":{"doc://SwiftPy/documentation/SwiftPy/PythonError/AssertionError(_:)":{"kind":"symbol","abstract":[],"title":"PythonError.AssertionError(_:)","role":"symbol","type":"topic","url":"\/documentation\/swiftpy\/pythonerror\/assertionerror(_:)","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/AssertionError(_:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AssertionError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}]},"doc://SwiftPy/documentation/SwiftPy":{"role":"collection","url":"\/documentation\/swiftpy","abstract":[{"text":"A fast and lightweight Python interpreter with Swift macro binding tools.","type":"text"}],"type":"topic","title":"SwiftPy","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","kind":"symbol"},"doc://SwiftPy/documentation/SwiftPy/PythonError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"PythonError","kind":"identifier"}],"kind":"symbol","abstract":[],"navigatorTitle":[{"text":"PythonError","kind":"identifier"}],"type":"topic","url":"\/documentation\/swiftpy\/pythonerror","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError","role":"symbol","title":"PythonError"}}}